pkg/machines/libvirt-dbus.js

```diff
@@ -1368,6 +1375,48 @@ export function getAllInterfaces(dispatch, connectionName) {
             .fail(ex => console.warn('getAllInterfaces action failed:', JSON.stringify(ex)));
 }
 
+function initInterfaces(connectionName) {
```

Comment:
I don't really like the fact that  your are duplicating so much code here.
All these function are essentially the getAll(Resource) functions just without calling the get(Resource) for parsing the XML.
I think it would have been much cleaner to actually pass a parameter to the original functions.
I don't mind having it as a fix up in the future though.

Reply:
Yes, please generalize this to `initResource(connectionName, method, updateCallback)`